rtorrent/src/main.cc

```diff
@@ -647,7 +647,7 @@ do_panic(int signum) {
 
 void
 print_help() {
-  std::cout << "Rakshasa's BitTorrent client version " VERSION "." << std::endl;
+  std::cout << "This is stickz modified version 0.9.8-v5.0 of Rakshasa's BitTorrent client version " VERSION "." << std::endl;
```

Comment:
Change to this:
`std::cout << "rtorrent stickz BitTorrent client version " STICKZ_VERSION "." << std::endl;`

Change this to link to the GitHub issues report section.
https://github.com/stickz/rtorrent/blob/6598f49fd548acaf44aaf102c1564c104b4c2e02/rtorrent/src/main.cc#L689

Add on Line 4: `AC_DEFINE(STICKZ_VERSION, "0.9.8-v5.0", rtorrent stickz version)`
https://github.com/stickz/rtorrent/blob/6598f49fd548acaf44aaf102c1564c104b4c2e02/rtorrent/configure.ac#L3-L4